index.js

```diff
@@ -224,8 +224,11 @@ function parseMetadata(dataset){
     $('#meta-title').html(title);
     var responsiblePartiesString = "";
     for (let i = 0; i < responsibleParties.length; i++) {
-        responsiblePartiesString += responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:individualName"] + ", " + 
-                                    responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:role"]["gmd:CI_RoleCode"] + "<br/>"
+        role = responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:role"]["gmd:CI_RoleCode"];
+        if ( role === 'principalInvestigator' )
+            role = 'Principal Investigator';
+        responsiblePartiesString += responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:individualName"] + " (" + 
+                                    role + ")<br/>";
     }
     $('#meta-responsibleParties').html(responsiblePartiesString.substring(0, responsiblePartiesString.length - 1));
```

Comment:
Not your mistake, but this should be `length - 5` to remove the whole `<br/>`.
```suggestion
     $('#meta-responsibleParties').html(responsiblePartiesString.substring(0, responsiblePartiesString.length - 5));
```